captum/_utils/common.py

```diff
@@ -18,6 +18,34 @@
 from torch.nn import Module
 
 
+def _parse_version(v: str, length: Optional[int] = 3) -> Tuple[int, ...]:
+    """
+    Parse version strings into tuples for comparison.
+
+    Versions should be in the form of "<major>.<minor>.<patch>", "<major>.<minor>",
+    or "<major>". The "dev", "post" and other letter portions of the given version will
+    be ignored.
+
+    Args:
+
+        v (str): A version string.
+        length (int, optional): The expected length of the output tuple. If the output
+            is less than the expected length, then it will be padded with 0 values. Set
+            to None for no padding or length checks.
+            Default: ``3``
+
+    Returns:
+        version_tuple (tuple of int): A tuple of integer values to use for version
+            comparison.
+    """
+    v = [n for n in v.split(".") if n.isdigit()]
+    assert v != []
+    if length is not None:
```

Comment:
A question: when will the length be useful ? If we always add 0s then it is not going to make big difference?
e.g. Whether we compare `(2,5) >= (2,4,5)` or `(2,5,0) >= (2,4,5)` we get the same outcome.

Reply:
I guess only in this case `(2,4,0) > (2,4)`  and `(2,4) > (2,4)` but is it likely ?

Reply:
@NarineK Yeah, I realize now that we don't need to pad the version tuple with zeros. If there are zeros in the version string, then they'll be used, otherwise we don't need to add them.

I've removed the length option from the function